nodestream/databases/neo4j/database_connector.py

```diff
@@ -19,6 +19,7 @@ def from_file_data(
         password: str,
         database_name: str = "neo4j",
         use_enterprise_features: bool = False,
+        apoc_iterate: bool = True,
```

Comment:
I wonder if at the connector level we should name this more generically. Like `use_apoc`. It could be a handy flag for other uses of the `apoc` library in the future. What do you think? Too generic?